charts/enterprise/prometheus/Chart.yaml

```diff
@@ -7,11 +7,11 @@ dependencies:
   - condition: exporters.enabled,exporters.node-exporter.enabled
     name: node-exporter
     repository: https://deps.truecharts.org
-    version: 1.0.35
+    version: 1.0.41
```

Comment:
you cannot do this  in the same PR as an edit of the dependency.
As the changed dependency isn't even released at this stage.

Actually: This is automated, please don't update dependency versions manually at all.